lib/after.js

```diff
@@ -130,6 +126,6 @@ test.after( function() {
 		config.get( 'closeBrowserOnComplete' ) === true ||
 		global.isHeadless === true
 	) {
-		return driverManager.quitBrowser( driver );
+		return await driverManager.quitBrowser( driver );
```

Comment:
since this is an async function, you can just return the promise from `driverManager.quitBrowser`. No need for the `await` here.

Reply:
Thanks, I wasn't sure about that but it makes sense